packages/protocol/contracts/ACE/ACE.sol

```diff
@@ -255,20 +398,21 @@ contract ACE is IAZTEC {
     function createNoteRegistry(
         address _linkedTokenAddress,
         uint256 _scalingFactor,
-        bool _canMint,
-        bool _canBurn,
+        bool _canAdjustSupply,
         bool _canConvert
     ) public {
         require(registries[msg.sender].flags.active == false, "address already has a linked note registry");
         NoteRegistry memory registry = NoteRegistry({
-            linkedToken: ERC20(_linkedTokenAddress),
+            linkedToken: IERC20(_linkedTokenAddress),
             scalingFactor: _scalingFactor,
             totalSupply: 0,
-            confidentialTotalSupply: bytes32(0x0),
+            confidentialTotalMinted: 0xdba4b8aad5b7a3f3e8e921ae22073db70b6d6590aface862af0d4eff2b920c9d,
```

Comment:
we should add a comment here describing what these are (notes of value 0x0, with a viewing key of 0x1)

Reply:
Yeah, the hashes aren't that descriptive!